src/Explorer/Panes/PublishNotebookPaneComponent.tsx

```diff
@@ -61,6 +71,39 @@ export class PublishNotebookPaneComponent extends React.Component<PublishNoteboo
       };
     };
 
+    this.takeScreenshot = (target: HTMLElement, onError: (error: Error) => void): void => {
+      const updateImageSrcWithScreenshot = (canvasUrl: string): void => {
+        this.props.onChangeImageSrc(canvasUrl);
+        this.setState({ imageSrc: canvasUrl });
+      };
+
+      target.scrollIntoView();
+      Html2Canvas(target, {
+        useCORS: true,
```

Comment:
Is there anything cross origin involved?

Reply:
this is for taking screenshots, there maybe a <img src=""> tag in a markdown which will need this to be captured